### PR TITLE
ToCode Solution: תרגול: מבוא לשורת הפקודה

### PR DESCRIPTION
**Pull request by Meiron**

### CODE REVIEW

File: 07_cmd_lab/cmdlab.txt [hunk 1]
```diff
@@ -1,13 +1,23 @@
-Intro To Command Line Lab.
+Intro To Command Line Lab - Amir Meiron
 Questions online: https://www.tocode.co.il/bundles/unix1/lessons/cmdlab
 
 1. Login to your UNIX account. How do you find:
   - your current shell?
+  echo $0 > bash
```
Comment (vitalipe) [inline 07_cmd_lab/cmdlab.txt:6]: administrative note: 
the > symbol has a very spacial meaning, as you'll learn soon.. 
I suggest to use the "#" symbol it means a comment 
(like // in many languages).

for example:
``` echo $0  # here you can put notes, and they are ignored  ```

File: 07_cmd_lab/cmdlab.txt [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
   - a list of all shells currently installed on the machine?
-
-2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". What information is held in the file? What are the fields in each line?
+  cat /etc/shells:
+	/bin/sh
+	/bin/dash
+	/bin/bash 
+	/bin/rbash
+2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". 
+What information is held in the file? - those are all the users in the system with their attributies.
```
Comment (vitalipe) [inline 07_cmd_lab/cmdlab.txt:16]: just as a side note: modern linux/unix systems storing password hashed in ```/etc/shadow```.. 

so passwords here are ignored (it's for backward compatibility)

File: 07_cmd_lab/cmdlab.txt [hunk 3]
```diff
@@ -1,13 +1,23 @@
...
+What are the fields in each line? login name : optional encrypted password : numerical user ID : numerical group ID : user name or comment field : user home directory : optional user command interpreter.
 
 3. In Unix, the /etc folder holds configuration files. Display all files in "/etc" folder ordered by file size (Hint: search in "man ls")
+cd \etc > ls -S -f
```
Comment (vitalipe) [inline 07_cmd_lab/cmdlab.txt:20]: almost... can you find  "-f" in the man page for ls?

Comment (Meiron) [reply 07_cmd_lab/cmdlab.txt:20]: yes,
# -f = do not sort

Comment (vitalipe) [reply 07_cmd_lab/cmdlab.txt:20]: right... but you *DO* want to sort the files don't you? :) 
so how would you fix it?

**Comment by vitalipe** [comment]
@Meiron - I'm closing this one.. feel free to fix your answer any time in the comments, open an issue and or/ contact me via PM on GitHub or directly by mail (vitali.pe@gmail.com).

**Comment by Meiron** [comment]
LS -S -l,

**Pull request closed without merging**